core/diffusion_ad.py

```diff
@@ -17,10 +17,10 @@ class DiffusionAD(BaseAlgo):
     anomaly_map_generator: ErrorMapGenerator
     anomaly_scorer: AnomalyScorer
 
-    def __init__(self, noiser, denoiser, anomaly_map_generator, anomaly_scorer, hparams):
+    def __init__(self, noiser, denoiser, anomaly_map_generator, anomaly_scorer, hparams, **kwargs):
         assert all(param in hparams for param in DIFFUSION_AD_REQUIRED_HPARAMS)
 
-        super().__init__(hparams)
+        super().__init__(hparams, **kwargs)
```

Comment:
interesting choice. 
usually i'm against implicit kwargs but in this use case i'll allow it.

Reply:
Its good for sifting up arguments meant for ancestors